modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/data-layout-builder/DataLayoutBuilderContextProvider.es.js

```diff
@@ -106,6 +107,15 @@ export default ({children, dataLayoutBuilder}) => {
 		dispatch({payload: {fieldTypes}, type: UPDATE_FIELD_TYPES});
 	}, [dataLayoutBuilder, dispatch]);
 
+	useEffect(() => {
+		const provider = dataLayoutBuilder.getLayoutProvider();
+		const onDrop = getHandlerDrop({dataDefinition, dataLayoutBuilder});
+
+		const eventHandler = provider.on('fieldDrop', onDrop);
```

Comment:
Now that we have https://github.com/matuzalemsteles/liferay-portal/blob/f0f090d53ee2e72b5396793a9f874782e0dcf654/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/hooks/usePage.es.js#L20 can we import and use those constants instead of hardcoding the string `'fieldDrop'`?
Also, the name `PAGE_TYPES` doesn't clearly say what it is. This is more like `PAGE_EVENT_TYPES` right? Or maybe it would feel more natural if it were like this:

```js
import {EVENT_TYPES} from 'dynamic-data-mapping-form-renderer';
```

Reply:
Oh yeah, `EVENT_TYPES` it will be clearer due to the context.